tensorflow/inference/buildspec.yml

```diff
@@ -0,0 +1,92 @@
+      *REPOSITORY_NAME ]
+context:
+  inference_context: &INFERENCE_CONTEXT
+    sagemaker_package_name:
+      source: docker/build_artifacts/sagemaker
+      target: sagemaker
+    init:
+      source: docker/build_artifacts/__init__.py
+      target: __init__.py
+    dockerd-entrypoint:
+      source: docker/build_artifacts/dockerd-entrypoint.py
+      target: dockerd-entrypoint.py
+    deep_learning_container:
+      source: ../../src/deep_learning_container.py
+      target: deep_learning_container.py
+
+images:
+ BuildEC2TensorflowCPUInferencePy3DockerImage:
+   <<: *INFERENCE_REPOSITORY
+   build: &TENSORFLOW_CPU_INFERENCE_PY3 false
+   image_size_baseline: &IMAGE_SIZE_BASELINE 4899
+   framework_version: &FRAMEWORK_VERSION 2.10.0
+   device_type: &DEVICE_TYPE cpu
+   python_version: &DOCKER_PYTHON_VERSION py3
+   tag_python_version: &TAG_PYTHON_VERSION py39
+   os_version: &OS_VERSION ubuntu20.04
+   tag: !join [ *FRAMEWORK_VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2" ]
+   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+   target: ec2
+   enable_test_promotion: true
+   context:
+     <<: *INFERENCE_CONTEXT
+ BuildEC2TensorflowGPUInferencePy3DockerImage:
+   <<: *INFERENCE_REPOSITORY
+   build: &TENSORFLOW_GPU_INFERENCE_PY3 false
+   image_size_baseline: &IMAGE_SIZE_BASELINE 10000
+   framework_version: &FRAMEWORK_VERSION 2.10.0
+   device_type: &DEVICE_TYPE gpu
+   python_version: &DOCKER_PYTHON_VERSION py3
+   tag_python_version: &TAG_PYTHON_VERSION py39
+   cuda_version: &CUDA_VERSION cu112
+   os_version: &OS_VERSION ubuntu20.04
+   tag: !join [ *FRAMEWORK_VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-ec2" ]
+   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+   target: ec2
+   enable_test_promotion: true
+   context:
+     <<: *INFERENCE_CONTEXT
+ BuildSageMakerTensorflowCPUInferencePy3DockerImage:
+   <<: *INFERENCE_REPOSITORY
+   build: &TENSORFLOW_CPU_INFERENCE_PY3 false
+   image_size_baseline: &IMAGE_SIZE_BASELINE 4899
+   framework_version: &FRAMEWORK_VERSION 2.10.0
+   device_type: &DEVICE_TYPE cpu
+   python_version: &DOCKER_PYTHON_VERSION py3
+   tag_python_version: &TAG_PYTHON_VERSION py39
+   os_version: &OS_VERSION ubuntu20.04
+   tag: !join [ *FRAMEWORK_VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker" ]
+   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+   target: sagemaker
+   enable_test_promotion: true
+   context:
+     <<: *INFERENCE_CONTEXT
+ BuildSageMakerTensorflowGPUInferencePy3DockerImage:
+   <<: *INFERENCE_REPOSITORY
+   build: &TENSORFLOW_GPU_INFERENCE_PY3 false
+   image_size_baseline: &IMAGE_SIZE_BASELINE 10000
+   framework_version: &FRAMEWORK_VERSION 2.10.0
```

Comment:
For inference images we define the `framework_version`. In the future, should we utilize the `version` variable ([here](https://github.com/aws/deep-learning-containers/pull/2433/files#diff-7f1162431d33417bf7b9927a6452ead707d1e5d3b24938a77534939071c76a87R4)) for the same?